server/server.go

```diff
@@ -11,6 +11,7 @@ import (
 	"syscall"
 	"time"
 
+	"github.com/NYTimes/gziphandler"
```

Comment:
I believe we will need an addition to Gopkg.toml since the gziphandler is not part of the project yet.

Otherwise a nice simple addition, sorry for letting it sit so long.

Reply:
Sorry for missing that, i've included the dependency. I think for now minor updates are ok, checking the state of the lib, outstanding PR's and velocity..their last update was 6 days ago, which seemed ok.